### ISSUE TITLE
Allow overwriting methods

### ISSUE THREAD

**Issue by thekid** [xp-lang/xp-records#1]
```php
// Uses generated toString()
record User(int $id, string $handle) {
}

// Defines its own toString() implementation
record User(int $id, string $handle) {
  public function toString() {
    return nameof($this).'(#'.$this->id.', handle= '.$this->handle).')';
  }
}
```

**Comment by thekid** [comment]
Currently yields:

```
Uncaught error: Compile error (Cannot redeclare T4::toString())
  at <source> [line 3 of dyn://T4]
  at <main>('src/test/php/') [line 0 of xp\unittest\TestRunner.class.php]
```

**Comment by thekid** [comment]
:shipit: Released in 0.3.0!
